mlir/test/mlir-miopen-driver/conv2d_host_validation.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: mlir-miopen-driver -fil_layout=yxck -in_layout=nhwc -out_layout=nhwk -pv -c | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s  
+// RUN: mlir-miopen-driver -fil_layout=yxck -in_layout=nhwc -out_layout=nhwk -pv -batchsize=128 -in_channels=8 -out_channels=128 -in_h=16 -in_w=16 -fil_h=5 -fil_w=5 -c | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s  
```

Comment:
Now that you have added new plumbing to dilation strides and padding please add new cases that alter those as well

Reply:
I added two test configurations with strides. I didn't find successful test cases with non-default dilations and paddings.

Reply:
Please document all failing conditions. Desirably use a separate issue each for strides/dilations/paddings, then reference them in this PR.

Reply:
I filed a new issue [#41](https://github.com/ROCmSoftwarePlatform/llvm-project-private/issues/41) for failed test cases. There are quite a few, so I put them in one issue.